generator/internal/stub_rest_generator.cc

```diff
@@ -152,12 +153,11 @@ Status StubRestGenerator::GenerateCc() {
   auto result = CcOpenNamespaces(NamespaceType::kInternal);
   if (!result.ok()) return result;
 
-  // TODO(#10214): Emit code to check options for a potential endpoint override.
   CcPrint(R"""(
 Default$stub_rest_class_name$::Default$stub_rest_class_name$(Options options)
     : rest_client_(rest_internal::MakePooledRestClient(
-          "https://$service_endpoint$", options)),
-      options_(std::move(options)) {}
+          options.get<EndpointOption>(), options)),
+          options_(std::move(options)) {}
```

Comment:
It looks like you restored the one line, but then added indentation to the next.